homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -0,0 +1,73 @@
+"""Suppoort for Amcrest IP camera binary sensors."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice, DEVICE_CLASS_MOTION)
+from homeassistant.const import CONF_NAME, CONF_BINARY_SENSORS
+from . import DATA_AMCREST, BINARY_SENSORS
+
+DEPENDENCIES = ['amcrest']
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=5)
+
+
+async def async_setup_platform(hass, config, async_add_devices,
+                               discovery_info=None):
+    """Set up a binary sensor for an Amcrest IP Camera."""
+    if discovery_info is None:
+        return
+
+    device_name = discovery_info[CONF_NAME]
+    binary_sensors = discovery_info[CONF_BINARY_SENSORS]
+    amcrest = hass.data[DATA_AMCREST][device_name]
+
+    amcrest_binary_sensors = []
+    for sensor_type in binary_sensors:
+        amcrest_binary_sensors.append(
+            AmcrestBinarySensor(amcrest.name, amcrest.device, sensor_type))
+
+    async_add_devices(amcrest_binary_sensors, True)
+    return True
```

Comment:
Nothing is checking this return value.

Reply:
Ok, I just modeled this after the other platform files (camera.py & sensor.py) that already existed. I'll remove the return statements from them as well.